website/client/components/shops/quests/index.vue

```diff
@@ -139,7 +139,7 @@
                     h4.popover-content-title(v-else) {{ item.text }}
                     .popover-content-text(v-if='item.locked && item.key === "lostMasterclasser1"') {{ `${$t('questUnlockLostMasterclasser')}` }}
                     .popover-content-text(v-if='item.locked && item.unlockCondition && item.unlockCondition.incentiveThreshold') {{ `${$t('loginIncentiveQuest', {count: item.unlockCondition.incentiveThreshold})}` }}
-                    .popover-content-text(v-if='item.locked && item.previous') {{ `${$t('unlockByQuesting', {title: item.previous})}` }}
+                    .popover-content-text(v-if='item.locked && item.previous && item.key !== "moon2" && item.key !=="moon3"') {{ `${$t('unlockByQuesting', {title: item.previous})}` }}
```

Comment:
could you move the condition to a method inside of this file, that way if any other quests have to handled like that, the checks are not hidden somewhere in the templates